test/command/11335.md

```diff
@@ -0,0 +1,109 @@
+Without fancy_lists, we get the Default counter styles
+
+```
+% pandoc -t native -f markdown_strict
+1. A
+2. B
+
+(a) A
+(b) B
+^D
+[ OrderedList
+    ( 1 , DefaultStyle , DefaultDelim )
+    [ [ Plain [ Str "A" ] ] , [ Plain [ Str "B" ] ] ]
+, Para
+    [ Str "(a)"
+    , Space
+    , Str "A"
+    , SoftBreak
+    , Str "(b)"
+    , Space
+    , Str "B"
+    ]
+]
+```
+
+and thus as HTML we don't have `style` or `type` in the `<ol>`:
+
+```
+% pandoc -t html -f markdown_strict
+1. A
+2. B
+
+(a) A
+(b) B
+^D
+<ol>
+<li>A</li>
+<li>B</li>
+</ol>
+<p>(a) A (b) B</p>
+```
+
+With fancy_lists, we get Decimal and Period instead of Default.
+
+```
+% pandoc -t native -f markdown_strict+fancy_lists
+1. A
+2. B
+
+(a) A
+(b) B
+^D
+[ OrderedList
+    ( 1 , Decimal , Period )
+    [ [ Plain [ Str "A" ] ] , [ Plain [ Str "B" ] ] ]
+, OrderedList
+    ( 1 , LowerAlpha , TwoParens )
+    [ [ Plain [ Str "A" ] ] , [ Plain [ Str "B" ] ] ]
+]
+```
+
+With commonmark, we get Decimal and Period even without fancy_lists.
+
+```
+% pandoc -t native -f commonmark
+1. A
+2. B
+
+(a) A
+(b) B
+^D
+[ OrderedList
+    ( 1 , Decimal , Period )
+    [ [ Plain [ Str "A" ] ] , [ Plain [ Str "B" ] ] ]
+, Para
+    [ Str "(a)"
+    , Space
+    , Str "A"
+    , SoftBreak
+    , Str "(b)"
+    , Space
+    , Str "B"
+    ]
+]
+```
+
+With fancy_lists and parens in the markdown source, we don't try to use a
+custom counter-style as the list-style-type in HTML output when no template is
+used.
```

Comment:
This is unusual -- the usual expectation is that the output you get without `-s` is the same you'll get with `-s` (save for the template).  Do we need this feature ?  What happens if we specify the custom list style type and it isn't defined in styles ?